src/townname.cpp

```diff
@@ -166,9 +166,9 @@ bool GenerateTownName(Randomizer &randomizer, uint32_t *townnameparts, TownNames
  * @param seed seed
  * @return seed transformed to a number from given range
  */
-static inline uint32_t SeedChance(uint8_t shift_by, int max, uint32_t seed)
+constexpr uint32_t SeedChance(uint8_t shift_by, size_t max, uint32_t seed)
```

Comment:
Seems weird to make these constexpr, even if it is possible, are they ever used that way?